src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -43,9 +42,17 @@ type CoreClinicalSchemaName =
   | ClinicalEntitySchemaNames.FOLLOW_UP
   | ClinicalEntitySchemaNames.SPECIMEN;
 
+const getCoreCompletionPercentage = (fields: CoreCompletionFields) =>
+  mean(Object.values(fields || {})) || 0;
+
+const getCoreCompletionDate = (donor: Donor, percentage: number) =>
+  percentage === 1
+    ? donor.completionStats?.coreCompletionDate || donor.updatedAt || new Date().toDateString()
```

Comment:
Is the logic here that if the file is missing the core completion date, you will just use the most recent updatedAt date for the value?

Reply:
yup. should I skip that and use the current time instead?

Reply:
discussed on slack - no